File: Python/백준/Bronze/8958. OX퀴즈/OX퀴즈.py
T = int(input())
while T>0:
    T -=1
    s = input()
    count = 0
    result = 0
    for i in range(len(s)):
        if s[i] == 'O':
            count += 1
            result += count
        else:
            count = 0
    print(result)

File: C++17/백준/Bronze/8958. OX퀴즈/OX퀴즈.cc
#include<iostream>
#include<string>
using namespace std;

int main() {
	int T;
	cin >> T;
	while(T--) {
		int count = 0, result=0;
		string ox;
		cin >> ox;

		for (int i = 0; i < ox.size(); i++) {
			if (ox[i] == 'O') {
				count++;
				result += count;
			}
			else
				count = 0;
		}
		cout << result << endl;
	}
}